app/build.sh

```diff
@@ -574,18 +575,27 @@ run_minishift() {
     if [ $(hasflag --full-reset) ] || [ $(hasflag --reset) ]; then
         # Only warning if minishift is not installed
         minishift delete
+        sleep 2
```

Comment:
Would it make sense to do a `minishift delete --clear-cache --force` would then this sleep be neccesary?

Reply:
Maybe not. Let me try that ...(oops forget to submit the comment ;-)